algorithms/index.html

```diff
@@ -92,15 +127,139 @@
                 </ul>
             </section>
 
+
+            <section>
+                <h3>Finding something to watch on TV v1</h3>
+                <ol class="left">
+                     <li>Turn on TV</li>
+                     <li>Watch what is on</li>
+                </ol>
+                <br><br>
+                <img src="images/tv_1_code.png" style="width:50%;">
+            </section>
+
+            <section>
+                <h3>Finding something to watch on TV v2</h3>
+                <ol class="left">
+                     <li>Turn on TV</li>
+                     <li>Flip through every channel and rate what is on</li>
+                     <li>Find the highest rated channel</li>
+                     <li>Watch</li>
+                </ol>
+            </section>
+
+            <section>
+                <img src="images/tv_2_code.png" style="width:80%;">
+            </section>
+
+             <section>
+                <h3>Finding something to watch on TV v3</h3>
+                <ol class="left">
+                     <li>Turn on TV</li>
+                     <li>Check 5 favorite channels and rate what is on</li>
+                     <li>Find the highest rated channel</li>
+                     <li>Watch</li>
+                </ol>
+            </section>
+
+            <section>
+                <img src="images/tv_3_code.png" style="width:80%;">
+            </section>
+
+             <section>
+                <h3>Which version is best?</h3>
+                <br>
+                <span class="">
+                     <p class="">Turn on the TV</p>
+                     <p class="">Rate all channels</p>
+                     OR
+                     <p class="">Rate top channels</p>
+                </span>
+            </section>
+
+            <!-- START OF DATA STRUCTURE -->
+            
+            <section>
+                <h1>Data Structures</h1>
+                <br>
+                <p>A way to store data <br>(so that it can be used as desired)</p>
+            </section>
+
+            <section>
+                <h2>Data Structures + Algorithms</h2>
+                <br>
+                <p>A good way to approach an algorithm is to think about the data structures that you know.</p>
+                <br>
+                <p>What are their properties, and how can they be used to your advantage?</p>
+            </section>
+
+            <section>
+                <h2>Example: Arrays/List</h2>
+                <img src="images/list.png">
+            </section>
+            <section>
+                <h2>Example: Sets</h2>
```

Comment:
I think folks who are used to JS might be confused about how to implement this since it doesn't directly tie into something they learned. Would be good to give a verbal explanation that some of these need to be implemented as part of the solution.